core/drivers/crypto/crypto_api/acipher/ecc.c

```diff
@@ -0,0 +1,412 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2018-2020 NXP
+ *
+ * Crypto ECC interface implementation to enable HW driver.
+ */
+#include <crypto/crypto_impl.h>
+#include <drvcrypt.h>
+#include <drvcrypt_acipher.h>
+#include <tee/tee_cryp_utl.h>
+#include <utee_defines.h>
+
+/*
+ * Returns the key size in bits for the given ECC curve
+ *
+ * @curve   ECC Curve ID
+ */
+static size_t get_ecc_keysize(uint32_t curve)
+{
+	size_t size_sec = 0;
```

Comment:
It looks like this "helper" variable just helps making this function longer than necessary.

Reply:
Ok, I'll remove it and return immediately instead breaking